### PR TITLE
separate  graphgen SDL file from graphql

### PR DESCRIPTION
**Pull request by dagda1**
## Motivation

Currently, the graphgen backstage SDL file is part of `@frontside/backstage-plugin-graphql` which ties it to this context.

We want to separate it to use it in other contexts.

## Approach

Create new `@frontside/graphgen-backstage` package.

### CODE REVIEW

**Comment by frontsidejack** [comment]
The following preview packages were published:
|                               Name                              |                               Version                              | Install Command                                                                                         |
| :-------------------------------------------------------------: | :----------------------------------------------------------------: | ------------------------------------------------------------------------------------------------------- |
|    <br><pre><code>@frontside/graphgen-backstage</code></pre>    | <br><pre><code>0.1.1-pc-package-graphgen-definition.1</code></pre> | <br><pre><code>yarn add @frontside/graphgen-backstage@pc-package-graphgen-definition</code></pre>       |
| <br><pre><code>@frontside/backstage-plugin-graphql</code></pre> | <br><pre><code>0.2.2-pc-package-graphgen-definition.1</code></pre> | <br><pre><code>yarn add @frontside/backstage-plugin-graphql@pc-package-graphgen-definition</code></pre> |
<p align='right'><em>Generated by <a href='https://github.com/thefrontside/actions/tree/main/publish-pr-preview'>@thefrontside/actions Frontside</a><em></p>

File: packages/graphgen/src/factory.ts [hunk 1]
```diff
@@ -80,13 +80,42 @@ const gen: Generate = (info) => {
   }
 }
 
+function createCacheStorage(filename: string): CacheStorage {
```
Comment (taras) [inline packages/graphgen/src/factory.ts]: @dagda1, on the client project we added [flat-cache](https://www.npmjs.com/package/flat-cache) should we use that here?

**Review by taras — APPROVED** [review]
👍

File: packages/graphgen/src/factory.ts (hunk 1 above)
Comment (dagda1) [reply packages/graphgen/src/factory.ts]: @taras we should, what branch is that on.

Comment (taras) [reply packages/graphgen/src/factory.ts]: I sent you a link in slack

Comment (dagda1) [reply packages/graphgen/src/factory.ts]: @taras done

**Pull request merged**